crates/goose/src/agents/default.rs

```diff
@@ -147,9 +147,20 @@ impl Agent for DefaultAgent {
             .expect("Failed to list systems")
     }
 
-    async fn passthrough(&self, _system: &str, _request: Value) -> SystemResult<Value> {
-        // TODO implement
-        Ok(Value::Null)
+    async fn passthrough(&self, system: &str, method: &str, params: Value) -> SystemResult<Value> {
+        let capabilities = self.capabilities.lock().await;
+        let client = capabilities
+            .get_system(system)
+            .await
+            .unwrap_or_else(|| panic!("System not found: {}", system));
```

Comment:
I think we should return an Err (SystemError::UnknownSystem) here instead of panicing?